x-pack/filebeat/module/cisco/amp/config/config.yml

```diff
@@ -28,7 +28,8 @@ request.transforms:
     target: url.params.limit
     value: {{ .limit }}
 request.rate_limit.limit: '[[ .last_response.header.Get "X-RateLimit-Limit" ]]'
-request.rate_limit.reset: '[[ .last_response.header.Get "X-RateLimit-Reset" ]]'
+request.rate_limit.early_limit: 1
+request.rate_limit.reset: '[[ add (toInt (.last_response.header.Get "X-RateLimit-Reset")) ((now).Unix) ]]'
```

Comment:
What documentation is being used as reference for these headers? I found https://api-docs.amp.cisco.com/api_resources?api_host=api.amp.cisco.com&api_version=v1 which uses `X-Rate-Limit-*` (note the dash in "Rate-Limit").

<img width="553" alt="Screen Shot 2022-06-22 at 15 07 09" src="https://user-images.githubusercontent.com/4565752/175116938-7dc5e46a-a4a2-46b8-9267-24c395a97c70.png">